pkg/solana/codec/byte_string_modifier.go

```diff
@@ -27,9 +27,9 @@ func (s SolanaAddressModifier) DecodeAddress(str string) ([]byte, error) {
 		return nil, fmt.Errorf("%w: failed to decode Base58 address: %s", commontypes.ErrInvalidType, err)
 	}
 
-	if pubkey.IsZero() {
-		return nil, fmt.Errorf("%w: zero-value address", commontypes.ErrInvalidType)
-	}
+	// if pubkey.IsZero() {
```

Comment:
Wanted to call this out if it was problematic. This interfered with interface tests when initializing accounts with an empty pubkey field. Don't think this is necessarily needed for the CCIP use case but is this a validation we need?